data/seeds/cli-example/policies/bridg-email-engine.json

```diff
@@ -6,9 +6,10 @@
   ],
   "effect": "allow",
   "resources": [
-    "rn:bridg:accounts:<[A-z0-9_-]+:brands:[A-z0-9_-]+:campaign-messaging(:.+)?>"
+    "rn:bridg:<(accounts:[0-9A-z_-]+:brands:[0-9A-z_-]+:)?campaign-messaging(.+)?>",
+    "rn:bridg:<(accounts:[0-9A-z_-]+:brands:[0-9A-z_-]+:)?audiences(.+)?>"
   ],
   "actions": [
-    "create", "delete", "read"
+    "create", "read", "update", "delete"
```

Comment:
Why is update added? for campaign-messaging or audiences?

Reply:
We need to add 'update' policy because email engine need to send PUT request to messaging service to update blast state